packages/esm-ward-app/src/routes.json

```diff
@@ -14,13 +14,23 @@
       }
      }
   },
-  "pages": [
+  "extensions": [
+    {
```

Comment:
Oh, one other thing I forgot to note... if merged in the, the Wards home page link will start appearing by default.  Not sure if we need to rather make this "opt-in"?

Reply:
RefApp-wise, I prefer links like this to be opt-out, so that our demos display as much functionality as possible.

Reply:
This app will use endpoints from bed management and emrapi.  I'm not sure this is something the refapp is prepared for yet.

Reply:
We should just add those modules into the RefApp. There's no real "preparation". The sooner we have them, the sooner we can start working on getting other parts of the app working with emrapi.

As a general rule, if frontend code is in the RefApp and it needs any backend changes, we should just update the RefApp.

Reply:
@ibacher  thanks! Should I ticket that (in the ref app project)?